docs/src/markdown/contributing.md

```diff
@@ -7,11 +7,11 @@ Sublime Versions | Description
 ST2\ <=\ version\ <\ ST3 | Supported on a separate branch, but not actively.  Any further fixes or enhancements must come from the community.  Issues for versions less than ST3 will not be addressed moving forward by me.  Pull requests are welcome for back-porting features, enhancements, or fixes to the old branch, but the content of the pull **must** already exist on the main, actively developed branch.  I will not allow an older branch to exceed the main branch in regards to functionality. |
 ST3 | Fully supported and actively maintained.
 
-Contribution from the community is encouraged and can be done in a variety of ways:
+Contributions from the community are encouraged and can be done in a variety of ways:
 
-- Bug reports.
-- Reviewing code.
-- Code patches via pull requests.
+- Bug reports,
+- Code reviewing,
+- Code patches via pull requests,
```

Comment:
Is this is a list, I find the commas odd.  Leaving off punctuation would have been less odd

Reply:
Sorry for "+" change, it was auto-introduced by the editor.